### PR TITLE
Oded/evaluator

### PR DESCRIPTION
**Pull request by odeds**

### CODE REVIEW

File: src/components/purchase/directives/purchaseOpenModal/purchaseOpenModal.drv.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    'use strict';
+
+    angular.module('znk.infra-web-app.purchase').directive('purchaseOpenModal',
+        function (purchaseService) {
+            'ngInject';
+
+            return {
+                restrict: 'A',
+                controller: function($element) {
+                    $element.bind('click', function() {
```
Comment (igormas) [inline src/components/purchase/directives/purchaseOpenModal/purchaseOpenModal.drv.js]: BInd is for senior, use on function.

File: src/components/purchase/directives/purchaseOpenModal/purchaseOpenModal.drv.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/**
+ * attrs:
+ */
+
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra-web-app.purchase').directive('purchaseOpenModal',
```
Comment (igormas) [inline src/components/purchase/directives/purchaseOpenModal/purchaseOpenModal.drv.js]: Change to component

File: src/components/evaluator/directives/evaluateReviewStates.directive.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/**
+ * evaluateReviewStates
+ *  ng-model: gets an object with activeState:Number(from EvaluatorStatesEnum)
```
Comment (igormas) [inline src/components/evaluator/directives/evaluateReviewStates.directive.js]: Lets discuss

File: src/components/evaluator/directives/evaluateResult.directive.js [hunk 4]
```diff
@@ -1,37 +1,25 @@
+/**
+ * evaluateResult
+ *   pointsGetter - get user current points
+ *   typeGetter - can be a subjectId or other type of id
+ */
 (function (angular) {
     'use strict';
+
     angular.module('znk.infra-web-app.evaluator').directive('evaluateResult',
```
Comment (igormas) [inline src/components/evaluator/directives/evaluateResult.directive.js]: directive -> component

File: src/components/purchase/directives/purchaseOpenModal/purchaseOpenModal.drv.js (hunk 1 above)
Comment (odeds) [inline src/components/purchase/directives/purchaseOpenModal/purchaseOpenModal.drv.js]: old school is the best.. changed.

File: src/components/purchase/directives/purchaseOpenModal/purchaseOpenModal.drv.js (hunk 2 above)
Comment (odeds) [inline src/components/purchase/directives/purchaseOpenModal/purchaseOpenModal.drv.js]: component restrict only to elements, so left it directive with restrict: A
changed name

File: src/components/evaluator/directives/evaluateResult.directive.js (hunk 4 above)
Comment (odeds) [inline src/components/evaluator/directives/evaluateResult.directive.js]: changed

File: src/components/evaluator/directives/evaluateReviewStates.directive.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+ *  and for evaluated state add points and type props for evaluate-result drv like:
+ *  {
+        activeState: EvaluatorStatesEnum.EVALUATED.enum,
+        points: 2.5,
+        type: 2
+    }
+ */
+(function (angular) {
+    'use strict';
+    angular.module('znk.infra-web-app.evaluator').directive('evaluateReviewStates',
```
Comment (odeds) [inline src/components/evaluator/directives/evaluateReviewStates.directive.js]: done

File: src/components/evaluator/directives/evaluateReviewStates.directive.js (hunk 3 above)
Comment (odeds) [inline src/components/evaluator/directives/evaluateReviewStates.directive.js]: changed name to evaluateQuestionReviewStates

**Pull request merged**